[2;32m2017-06-23 10:30:16,197 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://120.24.88.90:3306/fastfun?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-06-23 10:30:17,063 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2017-06-23 10:30:17,064 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2017-06-23 10:30:17,065 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2017-06-23 10:30:17,597 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-06-23 10:30:17,649 - INFO  [main:SelectorManager@37] - Creating 4 reactors...
[m[2;35m2017-06-23 10:30:17,902 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[7;31m2017-06-23 10:30:18,906 - ERROR [Xmemcached-Reactor-0:MemcachedConnector$InnerControllerStateListener@558] - Exception occured in controller
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2017-06-23 10:30:18,910 - ERROR [Xmemcached-Reactor-0:Reactor@325] - Reactor dispatch events error
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2017-06-23 10:30:18,909 - ERROR [main:XMemcachedClient@593] - connect to 127.0.0.1:11211 error
[mjava.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: no further information
	at com.google.code.yanf4j.core.impl.FutureImpl.get(FutureImpl.java:145)
	at net.rubyeye.xmemcached.XMemcachedClient.connect(XMemcachedClient.java:582)
	at net.rubyeye.xmemcached.XMemcachedClient.<init>(XMemcachedClient.java:870)
	at net.rubyeye.xmemcached.XMemcachedClientBuilder.build(XMemcachedClientBuilder.java:340)
	at com.juice.orange.game.cached.MemcachedResource.buildMemcached(MemcachedResource.java:34)
	at com.klw.fastfun.pay.data.ServerBootstrap.main(ServerBootstrap.java:68)
Caused by: java.net.ConnectException: Connection refused: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:404)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2017-06-23 10:30:18,913 - ERROR [main:XMemcachedClient@627] - Connect to 127.0.0.1:11211 fail
[mjava.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: no further information
	at com.google.code.yanf4j.core.impl.FutureImpl.get(FutureImpl.java:145)
	at net.rubyeye.xmemcached.XMemcachedClient.connect(XMemcachedClient.java:582)
	at net.rubyeye.xmemcached.XMemcachedClient.<init>(XMemcachedClient.java:870)
	at net.rubyeye.xmemcached.XMemcachedClientBuilder.build(XMemcachedClientBuilder.java:340)
	at com.juice.orange.game.cached.MemcachedResource.buildMemcached(MemcachedResource.java:34)
	at com.klw.fastfun.pay.data.ServerBootstrap.main(ServerBootstrap.java:68)
Caused by: java.net.ConnectException: Connection refused: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:404)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[2;32m2017-06-23 10:30:18,916 - INFO  [main:Application@25] - ÂàùÂßãÂåñAO...
[m[2;32m2017-06-23 10:30:18,944 - INFO  [main:ServerBootstrap@129] - Application PID:	2960
[m[2;35m2017-06-23 10:30:20,928 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 1 times
[m[7;31m2017-06-23 10:30:21,968 - ERROR [Xmemcached-Reactor-0:MemcachedConnector$InnerControllerStateListener@558] - Exception occured in controller
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2017-06-23 10:30:21,969 - ERROR [Xmemcached-Reactor-0:Reactor@325] - Reactor dispatch events error
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2017-06-23 10:30:21,972 - ERROR [Heal-Session-Thread:MemcachedConnector$SessionMonitor@169] - Reconnect to 127.0.0.1:11211 fail
[m[2;35m2017-06-23 10:30:23,974 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 2 times
[m[7;31m2017-06-23 10:30:25,008 - ERROR [Xmemcached-Reactor-0:MemcachedConnector$InnerControllerStateListener@558] - Exception occured in controller
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2017-06-23 10:30:25,009 - ERROR [Xmemcached-Reactor-0:Reactor@325] - Reactor dispatch events error
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2017-06-23 10:30:25,009 - ERROR [Heal-Session-Thread:MemcachedConnector$SessionMonitor@169] - Reconnect to 127.0.0.1:11211 fail
[m[2;35m2017-06-23 10:30:29,010 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 3 times
[m[7;31m2017-06-23 10:30:30,012 - ERROR [Xmemcached-Reactor-0:MemcachedConnector$InnerControllerStateListener@558] - Exception occured in controller
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2017-06-23 10:30:30,013 - ERROR [Xmemcached-Reactor-0:Reactor@325] - Reactor dispatch events error
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2017-06-23 10:30:30,012 - ERROR [Heal-Session-Thread:MemcachedConnector$SessionMonitor@169] - Reconnect to 127.0.0.1:11211 fail
[m[2;35m2017-06-23 10:30:36,013 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 4 times
[m[2;35m2017-06-23 10:30:36,032 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-06-23 19:16:50,116 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://120.24.88.90:3306/fastfun?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-06-23 19:16:51,040 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2017-06-23 19:16:51,041 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2017-06-23 19:16:51,041 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2017-06-23 19:16:51,581 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-06-23 19:16:51,647 - INFO  [main:SelectorManager@37] - Creating 4 reactors...
[m[2;35m2017-06-23 19:16:51,684 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-06-23 19:16:51,699 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-06-23 19:16:51,701 - INFO  [main:Application@25] - ÂàùÂßãÂåñAO...
[m[2;32m2017-06-23 19:16:51,724 - INFO  [main:ServerBootstrap@129] - Application PID:	9228
[m[2;32m2017-06-23 19:17:44,903 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://120.24.88.90:3306/fastfun?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-06-23 19:17:45,625 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2017-06-23 19:17:45,625 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2017-06-23 19:17:45,626 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2017-06-23 19:17:46,052 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-06-23 19:17:46,122 - INFO  [main:SelectorManager@37] - Creating 4 reactors...
[m[2;35m2017-06-23 19:17:46,147 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-06-23 19:17:46,165 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-06-23 19:17:46,167 - INFO  [main:Application@25] - ÂàùÂßãÂåñAO...
[m[2;32m2017-06-23 19:17:46,206 - INFO  [main:ServerBootstrap@129] - Application PID:	4524
[m[7;31m2017-06-23 19:18:58,901 - ERROR [Xmemcached-Reactor-2:MemcachedHandler@138] - XMemcached network layout exception
[mjava.io.IOException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at com.google.code.yanf4j.nio.impl.NioTCPSession.readFromBuffer(NioTCPSession.java:203)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onRead(AbstractNioSession.java:196)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onEvent(AbstractNioSession.java:341)
	at com.google.code.yanf4j.nio.impl.SocketChannelController.dispatchReadEvent(SocketChannelController.java:56)
	at com.google.code.yanf4j.nio.impl.NioController.onRead(NioController.java:157)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:295)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[2;35m2017-06-23 19:18:58,945 - WARN  [Xmemcached-Reactor-2:MemcachedConnector@365] - Remove a session: 127.0.0.1:11211
[m[2;35m2017-06-23 19:19:00,951 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 1 times
[m[7;31m2017-06-23 19:19:01,950 - ERROR [Xmemcached-Reactor-0:MemcachedConnector$InnerControllerStateListener@558] - Exception occured in controller
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2017-06-23 19:19:01,951 - ERROR [Xmemcached-Reactor-0:Reactor@325] - Reactor dispatch events error
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2017-06-23 19:19:01,950 - ERROR [Heal-Session-Thread:MemcachedConnector$SessionMonitor@169] - Reconnect to 127.0.0.1:11211 fail
[m[2;35m2017-06-23 19:19:03,954 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;35m2017-06-23 19:19:03,956 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 2 times
[m[2;35m2017-06-23 19:19:41,186 - WARN  [Thread-4:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :102, action:com.klw.fastfun.pay.common.action.CommonAction, method:getOtherPayCodes
[m[2;35m2017-06-23 19:19:41,890 - WARN  [Thread-22:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :16, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[7;31m2017-06-23 19:22:09,600 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ on [id: 0x96b16d95, /127.0.0.1:64410 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-06-23 19:22:09,605 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ on [id: 0x7a502e4c, /127.0.0.1:64411 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-06-23 19:22:09,609 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ on [id: 0xbb15bd55, /127.0.0.1:64412 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2017-06-23 19:22:34,070 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://120.24.88.90:3306/fastfun?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-06-23 19:22:34,723 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2017-06-23 19:22:34,724 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2017-06-23 19:22:34,724 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2017-06-23 19:22:35,034 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-06-23 19:22:35,080 - INFO  [main:SelectorManager@37] - Creating 4 reactors...
[m[2;35m2017-06-23 19:22:35,092 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-06-23 19:22:35,108 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-06-23 19:22:35,109 - INFO  [main:Application@25] - ÂàùÂßãÂåñAO...
[m[2;32m2017-06-23 19:22:35,126 - INFO  [main:ServerBootstrap@129] - Application PID:	10072
[m[2;35m2017-06-23 19:22:39,438 - WARN  [Thread-32:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :124, action:com.klw.fastfun.pay.common.action.CommonAction, method:getOtherPayCodes
[m[2;35m2017-06-23 19:22:40,052 - WARN  [Thread-13:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :25, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[7;31m2017-06-23 19:23:57,216 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ on [id: 0xca664694, /127.0.0.1:64478 :> /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-06-23 19:23:57,227 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ on [id: 0x97173dad, /127.0.0.1:64479 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-06-23 19:23:57,240 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ on [id: 0xbb187082, /127.0.0.1:64480 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2017-06-23 19:24:06,058 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://120.24.88.90:3306/fastfun?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-06-23 19:24:06,688 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2017-06-23 19:24:06,688 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2017-06-23 19:24:06,689 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2017-06-23 19:24:06,990 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-06-23 19:24:07,034 - INFO  [main:SelectorManager@37] - Creating 4 reactors...
[m[2;35m2017-06-23 19:24:07,053 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-06-23 19:24:07,067 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-06-23 19:24:07,069 - INFO  [main:Application@25] - ÂàùÂßãÂåñAO...
[m[2;32m2017-06-23 19:24:07,085 - INFO  [main:ServerBootstrap@129] - Application PID:	5504
[m[2;35m2017-06-23 19:24:11,369 - WARN  [Thread-32:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :136, action:com.klw.fastfun.pay.common.action.CommonAction, method:getOtherPayCodes
[m[2;35m2017-06-23 19:24:12,024 - WARN  [Thread-12:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :29, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[7;31m2017-06-23 19:26:17,856 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ on [id: 0xf38f8052, /127.0.0.1:64538 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-06-23 19:26:17,898 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ on [id: 0xd09838ca, /127.0.0.1:64540 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-06-23 19:26:17,859 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ on [id: 0x0aa6f107, /127.0.0.1:64539 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2017-06-23 19:26:28,305 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://120.24.88.90:3306/fastfun?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-06-23 19:26:29,072 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2017-06-23 19:26:29,073 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2017-06-23 19:26:29,073 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2017-06-23 19:26:29,523 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-06-23 19:26:29,580 - INFO  [main:SelectorManager@37] - Creating 4 reactors...
[m[2;35m2017-06-23 19:26:29,594 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-06-23 19:26:29,608 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-06-23 19:26:29,609 - INFO  [main:Application@25] - ÂàùÂßãÂåñAO...
[m[2;32m2017-06-23 19:26:29,627 - INFO  [main:ServerBootstrap@129] - Application PID:	9604
[m[2;35m2017-06-23 19:26:34,311 - WARN  [Thread-13:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :146, action:com.klw.fastfun.pay.common.action.CommonAction, method:getOtherPayCodes
[m[2;35m2017-06-23 19:26:34,914 - WARN  [Thread-33:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :29, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[7;31m2017-06-23 19:26:59,136 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ on [id: 0x9ebc26d2, /127.0.0.1:64607 :> /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-06-23 19:26:59,150 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ on [id: 0x1099a003, /127.0.0.1:64606 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-06-23 19:26:59,158 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ on [id: 0x0a949a86, /127.0.0.1:64608 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: ËøúÁ®ã‰∏ªÊú∫Âº∫Ëø´ÂÖ≥Èó≠‰∫Ü‰∏Ä‰∏™Áé∞ÊúâÁöÑËøûÊé•„ÄÇ
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2017-06-23 19:27:13,047 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://120.24.88.90:3306/fastfun?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-06-23 19:27:13,669 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2017-06-23 19:27:13,670 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2017-06-23 19:27:13,670 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2017-06-23 19:27:13,983 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-06-23 19:27:14,034 - INFO  [main:SelectorManager@37] - Creating 4 reactors...
[m[2;35m2017-06-23 19:27:14,047 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-06-23 19:27:14,062 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-06-23 19:27:14,065 - INFO  [main:Application@25] - ÂàùÂßãÂåñAO...
[m[2;32m2017-06-23 19:27:14,084 - INFO  [main:ServerBootstrap@129] - Application PID:	7012
[m[2;35m2017-06-23 19:27:17,158 - WARN  [Thread-32:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :145, action:com.klw.fastfun.pay.common.action.CommonAction, method:getOtherPayCodes
[m[2;35m2017-06-23 19:27:17,806 - WARN  [Thread-34:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :28, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m